### ISSUE TITLE
Docker suffering connectivity issues.

### ISSUE THREAD

**Issue by madeofpeople** [kalabox/kalabox#163]
http://status.docker.com

```
info: -- Step 15/18 --
info: Installing your Syncthing image...
info: DOCKER => Pulling image. name=kalabox/syncthing:v0.6.0, build=false, createOpts=undefined, startOpts=undefined, postProviderOpts=undefined
info: -- FAIL. --

HTTP code is 404 which indicates error: undefined - <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /images/create was not found on this server.</p>
</body></html>
```

**Comment by madeofpeople** [comment]
@pirog @bcauldwell note ```createOpts=undefined, startOpts=undefined, postProviderOpts=undefined```

**Comment by bcauldwell** [comment]
Thiago, ping me on slack when you get this and I'll try to troubleshoot
this for you.

On Sat, May 2, 2015 at 10:04 AM, Thiago de Mello Bueno <
notifications@github.com> wrote:

> @pirog <https://github.com/pirog> @bcauldwell
> <https://github.com/bcauldwell> note createOpts=undefined,
> startOpts=undefined, postProviderOpts=undefined
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kalabox/kalabox/issues/163#issuecomment-98380237>.
>



-- 
- Benjamin Cauldwell

**Comment by pirog** [comment]
TMB id check out https://github.com/kalabox/kalabox/issues/167 as i suspect
this is your issue as well.

On Sat, May 2, 2015 at 1:52 PM, bcauldwell <notifications@github.com> wrote:

> Thiago, ping me on slack when you get this and I'll try to troubleshoot
> this for you.
>
> On Sat, May 2, 2015 at 10:04 AM, Thiago de Mello Bueno <
> notifications@github.com> wrote:
>
> > @pirog <https://github.com/pirog> @bcauldwell
> > <https://github.com/bcauldwell> note createOpts=undefined,
> > startOpts=undefined, postProviderOpts=undefined
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/kalabox/kalabox/issues/163#issuecomment-98380237>.
> >
>
>
>
> --
> - Benjamin Cauldwell
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kalabox/kalabox/issues/163#issuecomment-98398782>.
>



-- 
Cheers,

Mike Pirog
Kalamuna
www.kalamuna.com

**Comment by pirog** [comment]
@madeofpeople 

We just released 0.8.0 which may resolve this issue. I would try uninstalling (https://github.com/kalabox/kalabox/wiki/Uninstalling-Kalabox) pulling in the new code and provisioning using 

```
npm install kalabox -g
kbox provision
```

If you did the developer install you should consult https://github.com/kalabox/kalabox/wiki/Updating-Kalabox

**Comment by madeofpeople** [comment]
hey, thans, 167 had solved the problem, I should have reported back. 
Feel free to close.
